data/sql/updates/pending_db_world/skinning.sql

```diff
@@ -0,0 +1,18 @@
+-- Create a new skinning loot with right percentage
+DELETE FROM `skinning_loot_template` WHERE (`Entry` = 70068) AND (`Item` IN (21887, 25649, 35229));
+INSERT INTO `skinning_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(70068, 21887, 0, 80, 0, 1, 1, 1, 1, 'Knothide Leather'),
+(70068, 25649, 0, 0, 0, 1, 1, 1, 3, 'Knothide Leather Scraps'),
```

Comment:
```suggestion
INSERT INTO `skinning_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
(70068, 21887, 0, 80, 0, 1, 1, 1, 1, 'Knothide Leather'),
(70068, 25649, 0, 0, 0, 1, 1, 2, 3, 'Knothide Leather Scraps'),
```
Sources say mincount should be 2. Just one last observation :)